library/src/main/java/org/apache/apex/malhar/lib/state/managed/ManagedTimeUnifiedStateImpl.java

```diff
@@ -138,6 +139,19 @@ public void purgeTimeBucketsLessThanEqualTo(long timeBucket)
     super.purgeTimeBucketsLessThanEqualTo(timeBucket);
   }
 
+  @Override
+  public void setup(Context.OperatorContext context)
+  {
+    // set UnBoundedTimeBucketAssigner to this managed state impl
+    if (timeBucketAssigner == null) {
+      UnBoundedTimeBucketAssigner unBoundedTimeBucketAssigner = new UnBoundedTimeBucketAssigner();
+      //unBoundedTimeBucketAssigner.setup(context);
```

Comment:
@siyuanh Do we need to change the default behavior of this managed state impl? Is it possible to set the required time bucket assigner on the managed state impl - similar to the change you made in the dedup operator?